pages/results.js

```diff
@@ -78,21 +63,10 @@ export default function Results({ data }) {
       <TripCountAndTravelMethods />
       <WorkArrangement workMode={data["work-mode"]} />
       <CommuteDays data={data} />
-
       <CommuteDistanceDistribution data={data} />
-      <Flex
-        border="2px solid red"
-        width="100%"
-        p="20px"
-        mt="20px"
-        direction="column"
-      >
-        Summary plus stats in horizontal bar chart
-        <Img
-          border="2px solid grey"
-          src="https://user-images.githubusercontent.com/88268603/174800361-a529ca2e-b257-489b-b8b8-4fa4e2a59b90.png"
-        />
-      </Flex>
+      <DistanceTravelledMode
+        distanceTravelledMode={data["distance-travelled-by-mode"]}
```

Comment:
hey @hqtan, small one, need to change the prop on results page as well.

Reply:
oh derp, I missed this, and broke deployment preview 🤦🏽 
Thx for picking this up @OlaShabalina !
🙇🏽